app/database_etl/postgres_tables_handler/table_generator.py

```diff
@@ -91,6 +106,12 @@ def create_multi_select_tables(original_data, current_time):
     # Get countries for each state
     multi_select_tables_dict["state"] = add_latlng_to_df("region", "state_name", multi_select_tables_dict["state"])
     multi_select_tables_dict["city"] = add_latlng_to_df("place", "city_name", multi_select_tables_dict["city"])
+
+    # Delete country iso2 column, no longer needed
+    # Note: only need this temporarily, so fine to drop
+    multi_select_tables_dict["state"] = multi_select_tables_dict["state"].drop(columns=['country_iso2'])
```

Comment:
I don't understand this country_iso2 logic - why are we creating this column and then dropping it?

Reply:
the country_iso2 col is required for the `add_latlng_to_df` function

Reply:
I will add as a comment